src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        // required variables - are not class variables, because
+        // they do not reflect the state of the class
+        int supply = 0;
+        int buy = 0;
+
+        String[] lines = readFile(fromFileName).split(System.lineSeparator());
+
+        // calculating
+        for (String stringLine : lines) {
+            String[] parsingLine = stringLine.split(",");
+            int amount = Integer.parseInt(parsingLine[AMOUNT_INDEX]);
+            switch (parsingLine[OPERATION_TYPE_INDEX]) {
+                case "supply" :
+                    supply += amount;
+                    break;
+                case "buy" :
+                    buy += amount;
+                    break;
+                default:
+                    throw new RuntimeException("This type of operation is not supported");
+            }
```

Comment:
extract it into separate method too and make splitters and operationTypes a constants

Reply:
then we have to return two values from the method(buy, supply), and if we make them class fields, then we will have to create a new class object for each operation